test/geocel/vg/Vecgeom.test.cc

```diff
@@ -77,6 +78,10 @@ class VecgeomVgdmlTestBase : public VecgeomTestBaseImpl
   public:
     SPConstGeo build_geometry() const final
     {
+        using namespace celeritas::cmake;
+        cout << color_code('x') << "VecGeom v" << vecgeom_version << " ("
+             << vecgeom_options << ") VGDML" << color_code(' ') << endl;
```

Comment:
```suggestion
             << vecgeom_options << ") using VGDML" << color_code(' ') << endl;
```
to follow the pattern of the other message.

Reply:
Yep that was a typo.